src/api.js

```diff
@@ -50,7 +50,15 @@ export const postComments = (commentToPost, review_id) => {
         body: commentToPost.body,
       }),
     }
-  ).then((response) => {
-    return response.json();
+  ).then((res) => {
+    return res.json();
   });
 };
+
+export const  removeComment = (comment_id) => {
+  fetch(`https://nc-super-awesome-games.herokuapp.com/api/comments/${comment_id}`, {
+  method: 'DELETE'
+})
+  .then(res => res.json())
+  .catch(err => console.log(err));
```

Comment:
just be careful adding a catch in your api file, it will mean the promise will always resolve where you are invoking the api in the component